content/pages/constitution.md

```diff
@@ -200,21 +200,33 @@ Committee office, M3 1003.
     new committee members for the upcoming term shall be announced on the
     website and sent out to all affiliated mailing lists.
 
-2.  New volunteers can apply to be on the committee for the following term, and
+2.  *New volunteers* shall be defined as any individual who has never served on
+    the committee before, or has not served on the committee in the past four
+    terms. They can nominate themselves for positions on the next term's committee
+    via the call for new committee members, subject to the following process:
+, and
```

Comment:
NOOO omg I can't believe I missed these three lines. Will have to replace this commit in all the branches I forked off it >.<